### PR TITLE
add organization and library info to keep cards

### PR DESCRIPTION
**Pull request by camhashemi**
@leogrim @ryanpbrewster @andrewconner 

I didn't remove any fields, so the next step is to deploy, test, then refactor the front-end to consume this.

### CODE REVIEW

**Comment by maxov** [comment]
Could this wait until after #17081? I'm taking out `OrganizationCard` there, and I don't want more merge conflicts

**Comment by camhashemi** [comment]
np @gratimax

File: kifi-backend/modules/common/app/com/keepit/model/Library.scala [hunk 1]
```diff
@@ -344,8 +344,8 @@ case class BasicLibrary(id: PublicId[Library], name: String, path: String, visib
 }
 
 object BasicLibrary {
-  def apply(library: Library, owner: BasicUser, org: Option[Organization])(implicit publicIdConfig: PublicIdConfiguration): BasicLibrary = {
-    val path = LibraryPathHelper.formatLibraryPath(owner, org.map(_.handle), library.slug)
+  def apply(library: Library, owner: BasicUser, orgHandle: Option[OrganizationHandle])(implicit publicIdConfig: PublicIdConfiguration): BasicLibrary = {
+    val path = LibraryPathHelper.formatLibraryPath(owner, orgHandle, library.slug)
```
Comment (leogrim) [inline kifi-backend/modules/common/app/com/keepit/model/Library.scala:348]: It's weird to have a discrepancy here between how the owner and how the organization is treated. Why did you remove the full `Organization`?

Comment (leogrim) [inline kifi-backend/modules/common/app/com/keepit/model/Library.scala:348]: Oh I see, ok!

**Comment by leogrim** [comment]
:+1:

File: kifi-backend/modules/common/app/com/keepit/model/Library.scala (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/common/app/com/keepit/model/Library.scala:348]: If you have an `Organization`, you also have an `OrganizationHandle`, but you might only have an `OrganizationHandle` when creating a `BasicLibrary` (as in my case here).
